third-test.md

```diff
@@ -0,0 +1,29 @@
+# Third Test Pattern
+
+## Title
+
+_Third Test_
+
+## Problem
+
+Still none.
+
+## Context
+
+Dunno.
+
+## Forces
+
+Slight pull to the light side.
+
+## Solution
+
+Similar to _insert pattern link to [First Test](first-test.md)_
```

Comment:
I think you're doing it right. From my point of view it is OK. You will only link already existing and reviewed patterns. Otherwise you will leave a `todo` comment, or?